### DISCUSSION TITLE
Base connection height in Lite bins

### DISCUSSION THREAD

**Discussion by tsebaschtian** [kennetek/gridfinity-rebuilt-openscad#82] [Q&A]
![image](https://user-images.githubusercontent.com/97031774/222129278-90b3f745-6215-423c-a146-bfbbfe1d4780.png)

I have an issue that when i render a lite bin, the connection between the bases is only 1 layer height and it bridges poorly and leaves a hole in the print. Is there a Constant that i can use to change this? it doesn't seem to correlate to Bottom Thickness, or Wall thickness or any other value i tried.

**Comment by kennetek** [comment]
Ah, this looks like an issue. I will open an issue for it.
